pkg/webservice/handlers_test.go

```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
-	assert.NilError(t, err)
-	assert.DeepEqual(t, partitionQueuesDao.TemplateInfo.MaxResource, maxResource.DAOMap())
-
-	guaranteedResourcesConf := make(map[string]string)
-	guaranteedResourcesConf["memory"] = "400000"
-	guaranteedResources, err := resources.NewResourceFromConf(guaranteedResourcesConf)
-	assert.NilError(t, err)
-	assert.DeepEqual(t, partitionQueuesDao.TemplateInfo.GuaranteedResource, guaranteedResources.DAOMap())
+	assert.Equal(t, partitionQueuesDao.Properties[configs.ApplicationSortPolicy], policies.StateAwarePolicy.String())
+	assert.DeepEqual(t, partitionQueuesDao.TemplateInfo, &templateInfo)
+
+	// assert child root.a fields
+	assert.Equal(t, len(partitionQueuesDao.Children), 1)
+	child := &partitionQueuesDao.Children[0]
+	assert.Equal(t, child.QueueName, "root.a")
+	assert.Equal(t, child.Status, objects.Active.String())
+	assert.Equal(t, child.Partition, "")
+	assert.Equal(t, child.PendingResource == nil, true)
+	assert.DeepEqual(t, child.MaxResource, maxResource.DAOMap())
+	assert.DeepEqual(t, child.GuaranteedResource, guaranteedResource.DAOMap())
+	assert.Equal(t, child.AllocatedResource == nil, true)
+	assert.Equal(t, child.PreemptingResource == nil, true)
+	assert.DeepEqual(t, child.HeadRoom, maxResource.DAOMap())
+	assert.Equal(t, child.IsLeaf, false)
+	assert.Equal(t, child.IsManaged, true)
+	assert.Equal(t, child.Parent, configs.RootQueue)
+	assert.Equal(t, child.AbsUsedCapacity == nil, true)
+	assert.Equal(t, child.MaxRunningApps, uint64(0))
+	assert.Equal(t, child.RunningApps, uint64(0))
+	assert.Equal(t, child.CurrentPriority, configs.MinPriority)
+	assert.Equal(t, child.AllocatingAcceptedApps == nil, true)
+	assert.Equal(t, len(child.Properties), 1)
+	assert.Equal(t, child.Properties[configs.ApplicationSortPolicy], policies.StateAwarePolicy.String())
+	assert.DeepEqual(t, child.TemplateInfo, &templateInfo)
+
+	// assert child root.a.a1 fields
+	assert.Equal(t, len(partitionQueuesDao.Children[0].Children), 1)
+	child = &partitionQueuesDao.Children[0].Children[0]
+	assert.Equal(t, child.QueueName, "root.a.a1")
+	assert.Equal(t, child.Status, objects.Active.String())
+	assert.Equal(t, child.Partition, "")
+	assert.Equal(t, child.PendingResource == nil, true)
+	assert.DeepEqual(t, child.MaxResource, maxResource.DAOMap())
+	assert.DeepEqual(t, child.GuaranteedResource, guaranteedResource.DAOMap())
+	assert.Equal(t, child.AllocatedResource == nil, true)
+	assert.Equal(t, child.PreemptingResource == nil, true)
+	assert.DeepEqual(t, child.HeadRoom, maxResource.DAOMap())
+	assert.Equal(t, child.IsLeaf, true)
+	assert.Equal(t, child.IsManaged, true)
+	assert.Equal(t, child.Parent, "root.a")
+	assert.Equal(t, child.AbsUsedCapacity == nil, true)
+	assert.Equal(t, child.MaxRunningApps, uint64(0))
+	assert.Equal(t, child.RunningApps, uint64(0))
+	assert.Equal(t, child.CurrentPriority, configs.MinPriority)
+	assert.Equal(t, child.AllocatingAcceptedApps == nil, true)
```

Comment:
nit: `assert.Assert()` instead of `Equal()`

Reply:
Thanks for suggestion, already address comments in [9621f51](https://github.com/apache/yunikorn-core/pull/772/commits/9621f515c3aaedbbdf58803279ab78ce9d0f9ece), thanks !